fluentd/README.md

```diff
@@ -153,16 +153,6 @@ For containerized environments, see the [Autodiscovery Integration Templates][2]
 | `<INIT_CONFIG>`      | blank or `{}`                                                     |
 | `<INSTANCE_CONFIG>`  | `{"monitor_agent_url": "http://%%host%%:24220/api/plugins.json"}` |
 
-##### Log collection
-
-_Available for Agent versions >6.0_
-
-Collecting logs is disabled by default in the Datadog Agent. To enable it, see [Kubernetes log collection documentation][18].
-
-| Parameter      | Value                                                |
-| -------------- | ---------------------------------------------------- |
```

Comment:
Not Sure we want to remove this kind of documentation . @NBParis for more insight